lib/top_hangman/round.rb

```diff
@@ -1,9 +1,23 @@
+require_relative 'render'
+
 module TopHangman
   class Round
     attr_reader :word
 
     def initialize(word)
       @word = word
+      @end_of_round = false
+      @guesses = []
```

Comment:
Is `guesses` all the tries that the user has had in 1 round? Or is it all the tries the user has had in all rounds?

Reply:
In 1 round, tries that are mistakes and assertions